GiniVision/Classes/ReviewViewController.swift

```diff
@@ -193,6 +193,14 @@ public typealias ReviewScreenFailureBlock = (_ error: GiniVisionError) -> ()
         updateMinZoomScaleForSize(scrollView.bounds.size)
     }
     
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // On initialization imageView frame is (0,0) so the image needs to be centered
+        // inside the ScrollView when its size has changed
+        self.updateConstraintsForSize(scrollView.bounds.size)
```

Comment:
If it's about the view changing size, wouldn't a method like `didLayoutSubviews` be more suitable?

Also, "// On initialization imageView'**S** frame"

Reply:
You are right, I don't know why I decided to put it on the viewWillAppear method. Thank you! 👍